packages/cozy-procedures/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "babel-preset-cozy-app": "1.5.1",
+    "cozy-doctypes": "^1.49.3",
```

Comment:
Why not. I understand the why, but to be coherent we should put it as peerDep ? :d

Reply:
Damnit yes, I told it --peer.